frontend/src/pages/learn/index.tsx

```diff
@@ -247,9 +265,18 @@ const Learn: FC = () => {
                     <h2 className="mb-8 font-display text-2xl font-medium md:mb-16 md:rounded-3xl md:bg-[#f5f5f5] md:p-6 md:text-4xl md:text-primary-700">
                       {t('quiz.navigation.title')}
                     </h2>
-                    <div className="mb-5">{renderComponent()}</div>
+                    <div>{renderComponent()}</div>
+                  </DialogContent>
+                  <DialogActions className="block"
```

Comment:
Use `flex` to move the `Stepper` to the bottom of `DialogContent`. `Stepper` shouldn't be inside the `DialogActions` component.